jcl/src/jdk.management/share/classes/openj9/lang/management/OpenJ9DiagnosticsMXBean.java

```diff
@@ -0,0 +1,145 @@
+/*[INCLUDE-IF Sidecar17]*/
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+
+package openj9.lang.management;
+
+import java.lang.management.PlatformManagedObject;
+
+/**
+ * <p>
+ * This interface provides APIs to dynamically trigger Dump agents. APIs are also available to
+ * configure dump options.
+ * This MXBean reuses the methods in com.ibm.jvm.Dump API 
+ * <br>
+ * <table border="1">
+ * <caption><b>Usage example for the {@link OpenJ9DiagnosticsMXBean}</b></caption>
+ * <tr> <td>
+ * <pre>
+ * {@code
+ *   ...
+ *   try {
+ *      mxbeanName = new ObjectName("openj9.lang.management:type=OpenJ9Diagnostics");
+ *   } catch (MalformedObjectNameException e) {
+ *      // Exception Handling
+ *   }
+ *   try {
+ *      MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
+ *      if (false == mbeanServer.isRegistered(mxbeanName)) {
+ *         // OpenJ9DiagnosticsMXBean not registered
+ *      }
+ *      OpenJ9DiagnosticsMXBean diagBean = JMX.newMXBeanProxy(mbeanServer, mxbeanName, OpenJ9DiagnosticsMXBean.class);
+ *   } catch (Exception e) {
+ *      // Exception Handling
+ *   }
+ * }
+ * </pre></td></tr>
+ * </table>
+ */
+public interface OpenJ9DiagnosticsMXBean extends PlatformManagedObject {
+	/**
+	 * Reset the JVM dump options to the settings specified when the JVM was started removing any additional
+	 * configuration done since then. This method may throw a DumpConfigurationUnavailableException if the dump 
```

Comment:
I think this should be 'ConfigurationUnavailableException' instead of 'DumpConfigurationUnavailableException'.